Tests/ManualTests/ManualTestingApp/Singles/CameraPage.ux

```diff
@@ -102,7 +102,21 @@
 			);
 		};
 		
-		exports.checkPermissions = function()
+		exports.checkCameraPermissions = function()
+		{
+			Camera.checkPermissions().then(
+				function(res)
+				{
+					console.log("response: "+res)
+				}
+			).catch(
+				function(error){
+					console.log("error: "+error)
+				}
+			);
+		};
+
+    exports.checkPermissions = function()
```

Comment:
Hmm, seems something went wrong when I resolved the conflict here... Would you mind fixing it up? :pray: